### ISSUE TITLE
description match 

### ISSUE THREAD

**Issue by szgy66** [vcg-uvic/lf-net-release#34]
hello, I had run your code successfully, even use my datasets. I got much keypoints and corresponding descriptions. Just as in your paper, I want to establish the corresponding relationship between the two graphs, but you emphasize that  ratio test cannot be used, so I want to ask, is there any way to match the descriptor?Or how do you filter good matching feature points.

Thanks

**Comment by etrulls** [comment]
You can use the ratio test. Just try different values (on a validation set) instead of picking what was recommended by SIFT. You also probably want to use bidirectional matching.

**Comment by szgy66** [comment]
I had tried. But I found that result less than SIFT

**Comment by szgy66** [comment]
> You can use the ratio test. Just try different values (on a validation set) instead of picking what was recommended by SIFT. You also probably want to use bidirectional matching.

I had tried. But I found that result less than sift,can't get the result as shown in paper

**Comment by etrulls** [comment]
What dataset are you using? What models are you using?

**Comment by szgy66** [comment]
> What dataset are you using? What models are you using?

Random images taken from a camera. Is there any requirement for this image. And I used the pre_trained model.

**Comment by etrulls** [comment]
Ok, I was confused because I thought you meant you couldn't reproduce the results in the paper. You meant it doesn't work well. I can't guess without looking at the images/results. Does it fail catastrophically, or does it just not look very good? How are you comparing it with SIFT, qualitatively? Are you using the rotation-invariant models if your data contains rotations, and the rotation-agnostic models otherwise?

**Comment by szgy66** [comment]
> Ok, I was confused because I thought you meant you couldn't reproduce the results in the paper. You meant it doesn't work well. I can't guess without looking at the images/results. Does it fail catastrophically, or does it just not look very good? How are you comparing it with SIFT, qualitatively? Are you using the rotation-invariant models if your data contains rotations, and the rotation-agnostic models otherwise?

It is effective, but compared with SIFT, its performance is not so good, for complex scenes, it and SIFT are both failed. I just simply observe the result of feature point matching, as shown in Figure 3 in the paper

**Comment by etrulls** [comment]
I'm afraid LF-Net is a bit dated, and it probably wasn't trained on your kind of data. I'd suggest you try something like SuperPoint, R2D2, or DISK, depending on the application.

**Comment by szgy66** [comment]
> I'm afraid LF-Net is a bit dated, and it probably wasn't trained on your kind of data. I'd suggest you try something like SuperPoint, R2D2, or DISK, depending on the application.

ok, thanks a lot

**Comment by ericzzj1989** [comment]
@szgy66 
Hello, have you worked out the custom datasets for training?
